lms/djangoapps/teams/csv.py

```diff
@@ -292,18 +300,17 @@ def validate_user_assignment_to_team_and_teamset(self, row):
                     return False
                 else:
                     self.user_ids_by_teamset_id[teamset_id].add(user.id)
-                    continue
+                    #continue
             max_team_size = self.course.teams_configuration.default_max_team_size
-            if max_team_size is not None and team.users.count() >= max_team_size:
-                if self.add_error_and_check_if_max_exceeded('Team ' + team.team_id + ' is full.'):
+            key = '{}.{}'.format(teamset_id, team_name)
```

Comment:
This constructed, string based ID feels like a hacky way to do it... Why not a data structure that more closely mirrors the team/teamset hierarchy?

Reply:
What data structure would you have in mind? I will switch to using a tuple of (teamset_id,team_name) but I don't see any benefit of a data structure and can't think of one that would make sense.